benchmark_runner/common/ocp_resources/create_ocp_resource_operations.py

```diff
@@ -93,7 +93,7 @@ def apply_patch(self, namespace: str, resource: str):
             check_approved = f"oc get InstallPlan -n {namespace} {name} -ojsonpath={{..spec.approved}}"
             approved = self.__oc.run(cmd=check_approved)
             # APPROVED false - need to patch
-            if not approved:
+            if 'false' == approved:
```

Comment:
It might be better to have __oc.run() do the conversion to boolean, at least as an option.  At the very least, use a case-insensitive comparison in case this changes again.

Reply:
__oc.run() just run the command, not handle the result.
Do you think we need to force shifting to boolean ?